3-enrich/config/iglu_resolver.json

```diff
@@ -12,6 +12,16 @@
             "uri": "http://iglucentral.com"
           }
         }
+      },
+      {
+        "name": "Iglu Central - Mirror 01",
```

Comment:
I think having `GCP` somewhere in name would make much more sense.

Reply:
Good point - let's rename to "Iglu Central - GCP Mirror"